script.js

```diff
@@ -0,0 +1,50 @@
+const bookTitle = document.getElementById('title');
+const bookAuthor = document.getElementById('author');
+const addBookButton = document.getElementById('addBook');
+
+const booksDiv = document.getElementById('booksDiv');
+
+const listOfBooks = [];
+
+function generateListOfBooks(arr) {
+  let items = '';
+  for (let i = 0; i < arr.length; i += 1) {
+    items += `
+        <li>${arr[i].title}</li> <br />
+        <li>${arr[i].author}</li> <br />
+        <li><button class="removeBtn" onclick="removeBook(${i})">Remove</button></li>
+        <hr />
+        `;
+  }
+  return items;
+}
+
+function showBooks() {
+  document.querySelector('form').reset();
+  localStorage.setItem('list of Books', JSON.stringify(listOfBooks));
```

Comment:
- [ ] Please make sure data is preserved in the browser's memory by using [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md) because when reloading the page manually the list of books keeps disappearing. Kindly take a look at the gif below to see and fix it please 👇 

https://user-images.githubusercontent.com/67087664/160861968-03d0337c-dd91-4b41-845e-6e50276c5890.mp4